ocw-cli/cli_app.py

```diff
@@ -1232,11 +1180,11 @@ def settings_screen(header):
          if option == '4':
               screen.addstr(25, x/2, "Enter Spatial Gridding Option [Model, Observation or User]:")
               new_spatial_grid_option = screen.getstr()
-              if new_spatial_grid_option.lower() == 'model':
-                   spatial_grid_option = 'Model'
-                   spatial_grid_setting = model_lat_res
-                   note = "Spatial gridding option has changed successfully to {0}".format(spatial_grid_option)
-              elif new_spatial_grid_option.lower() == 'observation':
+              #if new_spatial_grid_option.lower() == 'model':
```

Comment:
If the functionality is already in here why are we removing it? That seems a bit odd doesn't it? If we need to allow the user to select between multiple models when deciding to do this then that sounds like another ticket should be opened to add that functionality instead of just deleting existing functionality.